### PR TITLE
feat(useEyeDropper): add initialValue option, improve types

### PR DESCRIPTION
**Pull request by okxiaoliang4**

### CODE REVIEW

File: packages/core/useEyeDropper/index.ts [hunk 1]
```diff
@@ -1,14 +1,28 @@
 import { ref } from 'vue-demi'
 
-export function useEyeDropper() {
+type WindowEyeDropper = Window & typeof globalThis & {
```
Comment (antfu) [inline packages/core/useEyeDropper/index.ts:3]: Why don't we just type `EyeDropper` and also make it available in the return type?

File: packages/core/useEyeDropper/index.ts [hunk 2]
```diff
@@ -1,14 +1,28 @@
...
+  }
+}
+
+/**
+ * Reactive [EyeDropper API](https://developer.mozilla.org/en-US/docs/Web/API/EyeDropper_API)
+ *
+ * @see https://vueuse.org/useEyeDropper
+ * @param initialValue string
+ */
+export function useEyeDropper(initialValue = '') {
```
Comment (antfu) [inline packages/core/useEyeDropper/index.ts:18]: Prefer an object option here for future extensiablity

File: packages/core/useEyeDropper/index.ts (hunk 1 above)
Comment (okxiaoliang4) [reply packages/core/useEyeDropper/index.ts:3]: like this?
```typescript
export interface EyeDropper {
  new(): {
    open: () => Promise<{ sRGBHex: string }>
    [Symbol.toStringTag]: 'EyeDropper'
  }
}
```

Comment (antfu) [reply packages/core/useEyeDropper/index.ts:3]: Yes

File: packages/core/useEyeDropper/index.ts (hunk 2 above)
Comment (okxiaoliang4) [reply packages/core/useEyeDropper/index.ts]: ok~

**Pull request merged**